apps/browser/src/_locales/en/messages.json

```
@@ -2895,5 +2895,33 @@
   "commonImportFormats": {
     "message": "Common formats",
     "description": "Label indicating the most common import formats"
+  },
+  "overrideDefaultBrowserAutofillTitle": {
+    "message": "Make Bitwarden your default password manager?",
+    "description": "Dialog title facilitating the ability to override a chrome browser's default autofill behavior"
+  },
+  "overrideDefaultBrowserAutofillDescription": {
+    "message": "Leaving this setting off may cause conflicts between the Bitwarden auto-fill menu and your browser’s.",
```

nit; we're inconsistently using smart quotes

```suggestion
    "message": "Leaving this setting off may cause conflicts between the Bitwarden auto-fill menu and your browser's.",
```